README.md

```diff
@@ -29,7 +29,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: nightfallDLP action step
-        uses: nightfallai/nightfall_dlp_action@v1.1.3
+        uses: nightfallai/nightfall_dlp_action@v1.1.4
```

Comment:
can you bump our dogfood version too https://github.com/nightfallai/nightfall_dlp_action/blob/master/.github/workflows/nightfalldlp.yml

Reply:
done!

Reply:
wait do I need to do this in a separate PR? I can only reference 1.1.4 after I merge this PR and tag 1.1.4

Reply:
looks like the build works after merging this PR 🥳 

https://github.com/nightfallai/nightfall_dlp_action/pull/53